airbyte-integrations/connectors/source-gainsight-px/source_gainsight_px/schemas/feature.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Feature Schema",
+  "additionalProperties": true,
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
+    },
+    "name": {
+      "type": ["null", "string"]
+    },
+    "type": {
+      "type": ["null", "string"],
+      "enum": ["FEATURE", "MODULE"]
+    },
+    "parentFeatureId": {
+      "type": ["null", "string"]
+    },
+    "propertyKey": {
+      "type": ["null", "string"]
+    },
+    "status": {
+      "type": ["null", "string"],
+      "enum": ["ACTIVATED", "DELETED"]
+    },
+    "featureLabels": {
+      "type": "object",
+      "properties": {
```

Comment:
Add additionalProperties true to make backward compatibility

Reply:
Done